Index: Server/src/main/java/DatabaseManager.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import java.sql.Connection;\r\nimport java.sql.DriverManager;\r\nimport java.sql.SQLException;\r\n\r\npublic class DatabaseManager {\r\n    private static final String URL = \"jdbc:postgresql://db.kpvooltkaebzdkinypzh.supabase.co:6543/postgres\";\r\n    private static final String USER = \"postgres.db.kpvooltkaebzdkinypzh\";\r\n    private static final String PASSWORD = \"RazvanSiVasi\";\r\n\r\n    public static Connection getConnection() throws SQLException {\r\n        try {\r\n            //load driver\r\n            Class.forName(\"org.postgresql.Driver\");\r\n            return DriverManager.getConnection(URL, USER, PASSWORD);\r\n        } catch (ClassNotFoundException e) {\r\n            throw new SQLException(\"Driver-ul PostgreSQL nu a fost găsit!\", e);\r\n        }\r\n    }\r\n}
===================================================================
diff --git a/Server/src/main/java/DatabaseManager.java b/Server/src/main/java/DatabaseManager.java
--- a/Server/src/main/java/DatabaseManager.java	(revision 019fb4cfba49980ef32b8b00efa137af88595bc8)
+++ b/Server/src/main/java/DatabaseManager.java	(date 1767525366339)
@@ -3,8 +3,8 @@
 import java.sql.SQLException;
 
 public class DatabaseManager {
-    private static final String URL = "jdbc:postgresql://db.kpvooltkaebzdkinypzh.supabase.co:6543/postgres";
-    private static final String USER = "postgres.db.kpvooltkaebzdkinypzh";
+    private static final String URL = "jdbc:postgresql://aws-1-eu-west-1.pooler.supabase.com:6543/postgres";
+    private static final String USER = "postgres.kpvooltkaebzdkinypzh";
     private static final String PASSWORD = "RazvanSiVasi";
 
     public static Connection getConnection() throws SQLException {
